docs/AdvancedUsage.md

```diff
@@ -0,0 +1,165 @@
+# Advanced Usage Documentation
+
+| Step-Name         | Parameters                                                                                                                                                                                                      | Return                                       |
```

Comment:
i expected something like a java [doc ](https://javadoc.jenkins.io/hudson/model/Hudson.html) or a unique implementation of that.
| Method | Description | Parameters | Returns |
|--------|-------------|------------|---------|
| `ttCheckPackage` | Run an ecu.test package  | - `testCasePath`: `String `- Description<br/> - `packageConfig`: [`PackageConfig`](#packageconfig)  - description <br/> - `analysisConfig`: [`AnalysisConfig`](#analysisconfig) - description| [`TestResult`](#testresult) - description |